llpc/test/shaderdb/PipelineVsFs_TestInOutPacking.pipe

```diff
@@ -1,6 +1,75 @@
 ; BEGIN_SHADERTEST
 ; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
-; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+; SHADERTEST-LABEL: {{^// LLPC}} location input/output mapping results (FS shader)
+; CHECK:      (FS) Input:  loc = 0, comp = 0 =>  Mapped = 0, 0
```

Comment:
Do lines with `CHECK` do anything if FileCheck is invoked with `-check-prefix=SHADERTEST`?

Reply:
You are right. I will add a new test to check this corner case.